src/components/tabs/demoToolbarTabs/readme.html

```diff
@@ -0,0 +1 @@
+<p>The Toolbar Tabs demo shows how toolbar can be used with tabs</p>
```

Comment:
This `readme.html` file is not needed. Would be great if you remove this file :)